### PR TITLE
Import remaining non-secret worker configuration

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

**Comment by meatballhat** [comment]
@Lyoness r?

File: aws-precise-staging-1/worker.env [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export TRAVIS_WORKER_BUILD_API_PARANOID=false
```
Comment (Lyoness) [inline aws-precise-staging-1/worker.env]: we always need a $PARANOID env var :heart_decoration:

**Comment by Lyoness** [comment]
My questions are less about the actual line changes and more again about overall organization.  Would it be helpful for me to read commit history and past PRs to get fully oriented?

**Comment by meatballhat** [comment]
@Lyoness Let's chat about that out of band and maybe incorporate it back into the README(s)

**Pull request merged**